src/main/java/cat/udl/eps/softarch/demo/domain/Request.java

```diff
@@ -16,6 +16,7 @@
 public class Request extends Batch {
     @Id
     private Long id;
-
     private ZonedDateTime lastUpdate;
+    @OneToOne
+    private Take take;
```

Comment:
Better to use the name we have in the entity diagram "takenBy"

Reply:
I assume that in this case you mean fulfilled.